src/constants.js

```diff
@@ -22,6 +25,8 @@ const quorumTypes = {
 
 const MASTERNODE_DASH_AMOUNT = 1000;
 
+const HOME_DIR_PATH = path.resolve(os.homedir(), '.dashmate');
```

Comment:
We have it already defined in the container https://github.com/dashevo/dashmate/blob/e468f94d1b9458ea5d1c429f0baa8073316ac419/src/createDIContainer.js#L81

We could probably define it here and use this constant in the container?

Reply:
OK, I have changed the DI container so it now gets the variable as a constant. Does that look right?

Reply:
So we don't allow to configure the dashmate home dir with `DASHMATE_HOME_DIR` env anymore?

Reply:
Right, this option is no longer available.

Reply:
Why just don’t put this logic here?

Reply:
Because if the value is potentially variable, then it is by definition not a constant anymore and doesn't belong in `constants.js`